instapipeline/QuantiusAnnotation.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module contains the QuantiusAnnotation class.
+"""
+
+from fishanno import BaseAnnotation
+
+import json
+import pandas as pd
+
+
+class QuantiusAnnotation(BaseAnnotation):
+    """ Implementation of _import_annotations
+    for annotations from Quantius
+    """
+
+    def __init__(self, json_filepath, img_filename):
```

Comment:
Please document the input arguments in a docstring.

Reply:
It looks like a couple of the other classes also need their `__init__` methods documented. Can you please go through them and update?